Samples.xcodeproj/project.pbxproj

```diff
@@ -1383,11 +1407,13 @@
 				D71099702A2802FA0065A1C1 /* DensifyAndGeneralizeGeometryView.SettingsView.swift in Sources */,
 				E004A6ED2849556E002A1FE6 /* CreatePlanarAndGeodeticBuffersView.swift in Sources */,
 				E041ABD7287DB04D0056009B /* SampleInfoView.swift in Sources */,
+				1C43BC822A43781200509BF8 /* SetVisibilityOfSubtypeSublayerView.Model.swift in Sources */,
 				D752D9462A3A6F80003EB25E /* MonitorChangesToMapLoadStatusView.swift in Sources */,
 				E0082217287755AC002AD138 /* View+Sheet.swift in Sources */,
 				00181B462846AD7100654571 /* View+Alert.swift in Sources */,
 				E0FE32E728747778002C6ACA /* BrowseBuildingFloorsView.swift in Sources */,
 				D752D95F2A3BCE06003EB25E /* DisplayMapFromPortalItemView.swift in Sources */,
+				1CAB8D072A3CCBD8002AA649 /* SetVisibilityOfSubtypeFeatureLayerView.Views.swift in Sources */,
```

Comment:
It seems `SetVisibilityOfSubtypeFeatureLayerView.Views.swift` was added twice to the project. When the project does a self-clean it will be removed. cc @CalebRas